Frontend/ui-customer/src/Helpers/TipConfig.purs

```diff
@@ -73,6 +74,12 @@ hyderabadConfig variant =
     "TAXI_PLUS" -> mkTipConfig []
     _ -> mkTipConfig []
 
+chennaiConfig :: String -> TipConfig
```

Comment:
I think you should add a case for `BOOK_ANY` too, since user will be able to add auto tip using that.